askforhelp.rst

```diff
@@ -63,17 +63,16 @@ The big picture - An opening sentence should state the general problem that you
 
 **A snapshot of your environment** - For Plone and for other relevant products: provide version numbers. e.g., "I'm running Plone 5.0.4 under Python 2.7.11."
 
-**Steps to reproduce the issue** - Give information about your ideas of how this error appeared, what caused it or anything that could lead to reproducing the error on another computer. The expected result .
+**Steps to reproduce the issue** - Give information about your ideas of how this error appeared, what caused it or anything that could lead to reproducing the error on another computer, including your buildout.cfg, versions.cfg, version numbers of installed add-ons, detailed command lines, the complete error message stack. Mention your expected result.
 
-Asking help in real time chat (IRC)
------------------------------------
+Asking for help in online chat
+------------------------------
 
-Plone community real time chat is at the  `#plone <irc://irc.freenode.net/#plone>`_ IRC channel on the freenode IRC network.
+The Plone community uses `online chat <https://plone.org/support/chat>`_, specifically `Gitter <https://gitter.im/plone/public>`_ and traditional Plone IRC chat at the  `#plone <irc://irc.freenode.net/#plone>`_ IRC channel on the freenode IRC network.
 
```

Comment:
Next question why Gitter, there are also people on Slack so what is the idea behind adding Gitter but not Slack ?

IMHO I would be really careful with adding more and more options here, we should make it 'easy' for user and thsi means add the option(s) which the most people are using.
Otherwise you could end up in one channel which is not active at all.

So yeah I am not against it, but like I said we should be careful with splitting our own resources here.
It should not about what you or me personally like, it should be about which one is 'easy' and used by the most.